model/src/main/java/uk/gov/hmcts/payment/api/v1/model/govpay/GovPayAuthUtil.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.payment.api.v1.model.govpay;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.payment.api.v1.model.exceptions.PaymentInformationForbidden;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class GovPayAuthUtil {
+
+    @Value("${gov.pay.operational_services}")
+    private String[] operationalServices;
+
+    private List<String> operationalServicesList;
```

Comment:
how about below expression ..? so that other variable "operationalServicesList" can be avoided.
@Value("#{'${my.list.of.strings}'.split(',')}") 
private List<String> myList;